google/cloud/forseti/services/inventory/crawler.py

```diff
@@ -91,24 +91,20 @@ def get_client(self):
 
         return self.config.client
 
-    def on_child_error(self, resource, error):
+    def on_child_error(self, error):
         """Process the error generated by child of a resource
            Inventory does not stop for children errors but raise a warning
         """
 
         warning_message = '{}\n'.format(error)
-        if not resource._warning:
-            resource._warning = warning_message
-        else:
-            resource._warning += warning_message
         self.config.storage.warning(warning_message)
         self.config.progresser.on_warning(error)
 
     def update(self, resource):
-        """Update the error message in the row of the resource
+        """Update the row of an existed resource
```

Comment:
nit: do you mean ```existing``` rather than ```existed```?

Reply:
Can those files be left out of this PR and merged in post #836? In general I don't think there will be conflicts with my changes, but I'd like to understand the changes in context, and not in with the other unrelated changes in this PR.

Reply:
Thanks, I just checked dictionary. I mean "existing", let me fix this

Reply:
Sure. We can do that. I don't think there will be any conflicts though.